Bottom: 9dd8ba614fbbe9d82b591d8736ae29f0d34396a7
Top:    37d061895c83f91d3f517da6ad238319d308ee80
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2012-07-18 14:15:28 -0700

Refresh of dbg

---

diff --git a/src/cma.c b/src/cma.c
index 86dd787..9e3a891 100755
--- a/src/cma.c
+++ b/src/cma.c
@@ -48,6 +48,7 @@
 #include <byteswap.h>
 #include <stddef.h>
 #include <netdb.h>
+#include <stdio.h>
 
 #include "cma.h"
 #include <infiniband/driver.h>
@@ -139,6 +140,7 @@ static void ucma_cleanup(void)
 		free(cma_dev_array);
 		cma_dev_cnt = 0;
 	}
+	printf("ucma_cleanup\n");
 }
 
 static int check_abi_version(void)
@@ -212,6 +214,7 @@ int ucma_init(void)
 	if (cma_dev_cnt)
 		return 0;
 
+	printf("ucma_init\n");
 	pthread_mutex_lock(&mut);
 	if (cma_dev_cnt) {
 		pthread_mutex_unlock(&mut);
